Terminal.Gui/Core/Border.cs

```diff
@@ -306,16 +311,20 @@ public override bool MouseEvent (MouseEvent mouseEvent)
 		}
 
 		/// <summary>
-		/// Event to be invoked when any border property change.
+		/// Event to be invoked when any border property change except <see cref="Child"/>.
```

Comment:
This is how I would write this in terse, proper, English:

`Invoked when any property of Border changes (except <see cref="Child"/>).

Reply:
Done. Thanks.